subsys/usb/device_next/class/usbd_hid_macros.h

```diff
@@ -84,975 +84,15 @@
-#define HID_MPS_LESS_65_944 0
-#define HID_MPS_LESS_65_945 0
-#define HID_MPS_LESS_65_946 0
-#define HID_MPS_LESS_65_947 0
-#define HID_MPS_LESS_65_948 0
-#define HID_MPS_LESS_65_949 0
-#define HID_MPS_LESS_65_950 0
-#define HID_MPS_LESS_65_951 0
-#define HID_MPS_LESS_65_952 0
-#define HID_MPS_LESS_65_953 0
-#define HID_MPS_LESS_65_954 0
-#define HID_MPS_LESS_65_955 0
-#define HID_MPS_LESS_65_956 0
-#define HID_MPS_LESS_65_957 0
-#define HID_MPS_LESS_65_958 0
-#define HID_MPS_LESS_65_959 0
-#define HID_MPS_LESS_65_960 0
-#define HID_MPS_LESS_65_961 0
-#define HID_MPS_LESS_65_962 0
-#define HID_MPS_LESS_65_963 0
-#define HID_MPS_LESS_65_964 0
-#define HID_MPS_LESS_65_965 0
-#define HID_MPS_LESS_65_966 0
-#define HID_MPS_LESS_65_967 0
-#define HID_MPS_LESS_65_968 0
-#define HID_MPS_LESS_65_969 0
-#define HID_MPS_LESS_65_970 0
-#define HID_MPS_LESS_65_971 0
-#define HID_MPS_LESS_65_972 0
-#define HID_MPS_LESS_65_973 0
-#define HID_MPS_LESS_65_974 0
-#define HID_MPS_LESS_65_975 0
-#define HID_MPS_LESS_65_976 0
-#define HID_MPS_LESS_65_977 0
-#define HID_MPS_LESS_65_978 0
-#define HID_MPS_LESS_65_979 0
-#define HID_MPS_LESS_65_980 0
-#define HID_MPS_LESS_65_981 0
-#define HID_MPS_LESS_65_982 0
-#define HID_MPS_LESS_65_983 0
-#define HID_MPS_LESS_65_984 0
-#define HID_MPS_LESS_65_985 0
-#define HID_MPS_LESS_65_986 0
-#define HID_MPS_LESS_65_987 0
-#define HID_MPS_LESS_65_988 0
-#define HID_MPS_LESS_65_989 0
-#define HID_MPS_LESS_65_990 0
-#define HID_MPS_LESS_65_991 0
-#define HID_MPS_LESS_65_992 0
-#define HID_MPS_LESS_65_993 0
-#define HID_MPS_LESS_65_994 0
-#define HID_MPS_LESS_65_995 0
-#define HID_MPS_LESS_65_996 0
-#define HID_MPS_LESS_65_997 0
-#define HID_MPS_LESS_65_998 0
-#define HID_MPS_LESS_65_999 0
-#define HID_MPS_LESS_65_1000 0
-#define HID_MPS_LESS_65_1001 0
-#define HID_MPS_LESS_65_1002 0
-#define HID_MPS_LESS_65_1003 0
-#define HID_MPS_LESS_65_1004 0
-#define HID_MPS_LESS_65_1005 0
-#define HID_MPS_LESS_65_1006 0
-#define HID_MPS_LESS_65_1007 0
-#define HID_MPS_LESS_65_1008 0
-#define HID_MPS_LESS_65_1009 0
-#define HID_MPS_LESS_65_1010 0
-#define HID_MPS_LESS_65_1011 0
-#define HID_MPS_LESS_65_1012 0
-#define HID_MPS_LESS_65_1013 0
-#define HID_MPS_LESS_65_1014 0
-#define HID_MPS_LESS_65_1015 0
-#define HID_MPS_LESS_65_1016 0
-#define HID_MPS_LESS_65_1017 0
-#define HID_MPS_LESS_65_1018 0
-#define HID_MPS_LESS_65_1019 0
-#define HID_MPS_LESS_65_1020 0
-#define HID_MPS_LESS_65_1021 0
-#define HID_MPS_LESS_65_1022 0
-#define HID_MPS_LESS_65_1023 0
-#define HID_MPS_LESS_65_1024 0
 
 #define HID_MPS_LESS_65(x) UTIL_PRIMITIVE_CAT(HID_MPS_LESS_65_, x)
 
 /*
  * If all the endpoint MPS are less than 65 bytes, we do not need to define and
  * configure an alternate interface.
  */
-#define HID_ALL_MPS_LESS_65(n)							\
-	UTIL_AND(HID_MPS_LESS_65(DT_INST_PROP(n, out_report_size)),		\
+#define HID_ALL_MPS_LESS_65(n)									\
+	UTIL_AND(COND_CODE_1(HID_MPS_LESS_65(DT_INST_PROP_OR(n, out_report_size, 0)), (1), (0)),\
```

Comment:
The `out_report_size` has to be read as `DT_INST_PROP_OR(n, out_report_size, 0)` because `out_report_size` is optional and the absence of it should trigger the "1" branch.